README.md

```diff
@@ -45,18 +45,20 @@ Codecs supported for CD-DA emulation:
 | **WAV**        | Yes (built-in)             | Yes - SDL\_sound 1.2 (internal)<sup>[6],＊</sup>
 | **AIFF**       | No                         | Yes - SDL\_sound 1.2 (internal)<sup>[6],＊</sup>
 
-<sup>＊ - SDL 1.2 is not actively maintained any more.</sup>  
-<sup>† - 22.05 kHz, 44.1 kHz, 48 kHz; mono, stereo</sup>  
-<sup>‡ - 44.1 kHz stereo only</sup>  
-<sup>§ - SDL\_sound supports it, but the feature might be broken or DOSBox does not indicate support.</sup>  
+<sup>＊- SDL 1.2 was last updated 2013-08-17 and SDL\_sound 2008-04-20</sup>
+
+<sup>† - 22.05 kHz, 44.1 kHz, 48 kHz; mono, stereo</sup>
+
+<sup>‡ - 44.1 kHz stereo only</sup>
+
+<sup>§ - Broken or unsupported in either SDL\_sound or DOSBox</sup>
```

Comment:
Previously these lines used 2 spaces to break the line, which I presumed was correct markdown… Does it generate any warning? In this case, we don't want every annotation to be in a separate paragraph.

Reply:
Yes, separate paragraphs is unnecessary visual bloat.  Two (or more) spaces still generates:
- README.md:48: MD009 Trailing spaces
- README.md:49: MD009 Trailing spaces
- README.md:50: MD009 Trailing spaces

Edit: Interesting.. Markdown uses the more explicit "\" to create a line-break without starting a new paragraph, similar to <br>.  Will try that.

Reply:
Edit 2: \\ works!

Makes me think we can now safely add a "strip-trailing-whitespace" commit-hook, because nothing I know of should have trailing whitespace.. can you think of anything?

Reply:
Nope, breaking space was the only thing - and the new way is better than having 2 trailing spaces :)